acestep/models/mlx/dit_model.py

```
@@ -438,6 +438,7 @@ def __init__(
         layer_types: Optional[list] = None,
         rope_theta: float = 1_000_000.0,
         max_position_embeddings: int = 32768,
+        encoder_hidden_size: int = None,
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)